ex03.ex

```diff
@@ -76,8 +80,10 @@ defmodule Ex03 do
       true
 
   """
+   def list_contains([_h | _t], _a), do: true
```

Comment:
## 0
The first line will always match for any nonempty list:

~~~ elixir
iex> Ex03.list_contains([1,2,3,4], :wombat)
true
~~~